### PR TITLE
Removed bad faith duplicate for SWAD

### PR DESCRIPTION
**Pull request by Groctel**
Reopened from #1097 using the cleanup branch as suggested by the last comment (and also #1141 because I missclicked and fucked up). 

## Original request:

The original submission (https://www.reddit.com/r/placeAtlas2/comments/twq66p/submission_for_swad/) was discussed and approved from within the student community that created the SWAD logo. The removed submission (https://www.reddit.com/r/placeAtlas2/comments/tx2oa3/new_submission/) is from a third party actor operating in bad faith whose description isn't aligned with the ideals of the community and paints a wrong image about an official platform of the University.

### CODE REVIEW

**Comment by Codixer** [comment]
Please resolve the conflict

File: web/atlas.json [hunk 1]
```diff
@@ -3516,7 +3515,7 @@
...
-{"id": "tx27xv", "submitted_by": "InsaneByte", "name": "SWAD", "description": "SWAD is an educational platform of the University of Granada originally created for teaching, its creator Antonio Cañas Vargas, professor of computer engineering and computer architecture, began to add features that students requested and the students ended up turning it into a shitpost platform loved in the ETSIIT (technical school of computer engineering and telecommunications) of Granada.nA group of 40 students came together to create the r/place logo.n", "website": "https://swad.ugr.es/es", "subreddit": "", "center": [916.5, 1320.5], "path": [[901.5, 1315.5], [901.5, 1325.5], [931.5, 1325.5], [931.5, 1315.5], [901.5, 1315.5], [901.5, 1315.5]]},
+{"id": "tx27ur", "submitted_by": "bisc0tte_", "name": "Dakota", "description": "PearlescentMoon's cat!", "website": "https://youtube.fandom.com/wiki/PearlescentMoon", "subreddit": "/r/hermitcraft", "center": [899.5, 627.5], "path": [[897.5, 624.5], [901.5, 624.5], [901.5, 630.5], [897.5, 630.5], [897.5, 624.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3518]: This entry was modified by someone else, but your change brought back the old version. Remove this old version of the entry.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Resolve comments.

**Comment by AnonymousRandomPerson** [comment]
Looks like another merge conflict to resolve.

**Comment by Groctel** [comment]
It should be ok now XD

**Comment by AnonymousRandomPerson** [comment]
Seems the JSON validation failed now.

**Comment by Groctel** [comment]
I don't see the error in the diffs :/

**Comment by AnonymousRandomPerson** [comment]
In your latest changes, the file contains a ">>>>>>> 5649a6d814049831d8b8bc158a87e472dab13f1d" at the end. This is a merge conflict marker, and should be removed.

**Comment by Groctel** [comment]
I've pushed it as an amendment to the previous commit

**Pull request merged**